04-interpretable-models.Rmd

```diff
@@ -143,25 +143,25 @@ pretty_rownames = function(rnames){
 lm_summary_print = lm_summary
 rownames(lm_summary_print) = pretty_rownames(rownames(lm_summary_print))
 
-kable(lm_summary_print[,c('Estimate', 'Std. Error')], digits = 1)
+kable(lm_summary_print[,c('Weight estimate', 'Std. Error')], digits = 1)
```

Comment:
That is actually the name of the column in the R data.frame. Changing this would throw an error. Could you revert that line?

Reply:
How can the axis label be changed (cf. the next figure)?

Reply:
This changes the column names of the printed table:
```
kable(lm_summary_print[,c('Estimate', 'Std. Error')], digits = 1, col.names = c('Weight estimate', 'Std. Error'))
```
Just figured this out. I learned something new.